src/meta/src/barrier/mod.rs

```diff
@@ -400,6 +399,31 @@ where
                 .into()
             }
         });
+        try_join_all(inject_futures).await?;
+        let collect_futures = info.node_map.iter().filter_map(|(_, node)| {
+            let request_id = Uuid::new_v4().to_string();
+            let prev_epoch = command_context.prev_epoch.0;
+            async move {
+                let mut client = self.env.stream_client_pool().get(node).await?;
```

Comment:
If there's no actor to send for some compute node, should we still call `barrier_complete`?